README.md

```diff
@@ -68,15 +68,16 @@ User Manual
 __Description:__
 
 The Kion CLI allows users to perform common Kion workflows via the command
-line. Users can quickly generate short term access keys (stak) via configured
-favorites or by walking through an account and role selection wizard.
+line. Users can quickly generate short term access keys (stak) or federate
+into the cloud service provider web console via configured favorites or by
+walking through an account and role selection wizard.
 
 __Commands:__
 
 ```text
 stak, s            Generate short-term access keys.
 
-favorite, fav, f   Access pre-configured favorites to quickly generate staks.
+favorite, fav, f   Access pre-configured favorites to quickly generate staks or federate into the cloud service provider console depending on the access_type defined in the favorite.
```

Comment:
Code block lines don't wrap when rendered so let's add a line break manually to wrap.
```suggestion
favorite, fav, f   Access pre-configured favorites to quickly generate staks or federate
                          into the cloud service provider console depending on the access_type
                          defined in the favorite.
```